src/engines/TrainElectricEngine.hpp

```diff
@@ -21,8 +63,40 @@ namespace godot {
             bool get_converter_switch_pressed() const;
             void set_compressor_switch_pressed(bool p_state);
             bool get_compressor_switch_pressed() const;
+            void set_engine_power_source(int p_source);
+            int get_engine_power_source() const;
+            int get_number_of_collectors() const;
+            void set_number_of_collectors(int p_coll_no);
+            void set_max_voltage(float p_max_voltage);
+            float get_max_voltage() const;
+            void set_max_current(float p_max_current);
+            float get_max_current() const;
+            void set_max_collector_lifting(float p_max_collector_lifting);
+            float get_max_collector_lifting() const;
+            void set_min_collector_lifting(float p_min_collector_lifting);
+            float get_min_collector_lifting() const;
+            float get_csw() const;
+            void set_min_main_switch_voltage(float p_min_main_switch_voltage);
+            float get_min_main_switch_voltage() const;
+            void set_min_pantograph_tank_pressure(float p_min_pantograph_tank_pressure);
+            float get_min_pantograph_tank_pressure() const;
+            void set_max_pantograph_tank_pressure(float p_max_pantograph_tank_pressure);
+            float get_max_pantograph_tank_pressure() const;
+            void set_overvoltage_relay_value(bool p_overvoltage_relay_value);
```

Comment:
atrybut nazywa sie `overvoltage_relay` i jest typu bool, to nie dodawalbym suffixu `_value`. Sugeruje wartość z jakiegoś zakresu, a tu mamy tylko on/off. Może `overvoltage_relay_enabled` ?

Reply:
Mój błąd. Powinienem był zostawić set_overvoltage_relay